### DISCUSSION TITLE
新手入门，问个简单的问题：为什么在WebStorm里使用ElementPlus无提示？已将ElementPlus文件夹设置为library

### DISCUSSION THREAD

**Discussion by DongAndDi** [element-plus/element-plus#14059] [Feature Request]
Feature type: **`Other`**

## Details

- Enhanced component name: `All`

## Description

为什么在WebStorm里使用ElementPlus无提示？已将ElementPlus文件夹设置为library

### Additional comments

_(empty)_

**Comment by Zz-ZzzZ** [comment]
项目里新建一个`d.ts`然后引入它的声明文件就行，我也是这样莫名奇妙的，只能识别`web-type.json`内定义的组件

```
export type {} from 'element-plus/global'
```
